sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -445,7 +445,7 @@
           "id": "es4nl35"
         },
         {
-          "label": "What was your partner’s income (as shown on line 15000 of previous year’s Canada Revenue Agency tax return)? If they didn’t file taxes, enter total income from all sources in and outside of Canada.",
+          "label": "Enter your spouse/common-law partners reported total income from line 15000 of previous year income tax return. If they did not file taxes, enter their total income from all sources both inside and outside of Canada.",
```

Comment:
The following AC to update partner portal declaration is missing. 
![image](https://github.com/user-attachments/assets/77426dd8-1b4c-4ee0-946f-80238bca7e44)